cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -141,17 +141,17 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+start_addon /etc/kubernetes/addons/namespace.yaml 100 10 "" &
+
 # Wait for the default service account
```

Comment:
it's still correct but could be more explicit. There is one default serviceaccount per namespace and it is named "deafault" in all namespaces, not just the default namespace